inst/extdata/fitted_parameters.csv

```diff
@@ -1,13 +1,18 @@
-"parameter","value"
-"s_hosp_D",2
-"gamma_hosp_D",0.291115375275389
-"s_hosp_R",1
-"gamma_hosp_R",0.109204638515258
-"s_triage",1
-"gamma_triage",0.509957901224795
-"s_ICU_D",2
-"gamma_ICU_D",0.291386107435204
-"s_ICU_R",2
-"gamma_ICU_R",0.354142862957898
-"s_stepdown",1
-"gamma_stepdown",0.452381010083587
+"","parameter","value"
```

Comment:
Why do we have the first column?

Reply:
I've gotten rid of it, wasn't serving any purpose.